en/docs/references/error-catalog.md

```diff
@@ -216,3 +216,78 @@ This document describes all the REST API error codes that are used in WSO2 Ident
   </tbody>
 </table>
 </div>
+
+## Human Task Approval Errors
+
+<div class="errorcode">
+<table>
+  <thead>
+    <th>Error Code</th>
+    <th>HTTP Status Code</th>
+    <th>Error Message</th>
+    <th>Possible Cause</th>
+  </thead>
+  <tbody>
+      <tr>
+        <td>HTA-10001</a></td>
+        <td>403</td>
+        <td>Access Denied.</td>
+        <td>You are not authorized to perform this task.</td>
+      </tr>
+      <tr>
+        <td>HTA-10002</a></td>
+        <td>400</td>
+        <td>Invalid Input Provided.</td>
+        <td>The provided Task ID is not in the correct format.</td>
+      </tr>
+      <tr>
+        <td>HTA-10003</a></td>
+        <td>404</td>
+        <td>Task does not exists.</td>
+        <td> </td>
+      </tr>
+      <tr>
+        <td>HTA-10004</a></td>
+        <td>400</td>
+        <td>Invalid input data provided.</td>
+        <td> </td>
+      </tr>
+      <tr>
+        <td>HTA-10005</a></td>
+        <td>400</td>
+        <td>Unacceptable input provided.</td>
+        <td>Only [CLAIM, RELEASE, APPROVED, REJECTED] are acceptable.</td>
+      </tr>
+      <tr>
+        <td>HTA-10006</a></td>
+        <td>400</td>
+        <td>Unable to change the approval status.</td>
+        <td>Invalid state change is requested for the given task.</td>
+      </tr>
+      <tr>
+        <td>HTA-10007</a></td>
+        <td>400</td>
+        <td>Unable to update the approval status.</td>
+        <td>Invalid state change is requested for the given task.</td>
+      </tr>
+      <tr>
+        <td>HTA-15002</a></td>
+        <td>500</td>
+        <td>Unable to retrieve approvals for the user.</td>
+        <td>Server Encountered an error while retrieving approvals for user.</td>
+      </tr>
```

Comment:
can we make the "e" in "Encountered" simple case? Same for the next three error codes as well please.

Reply:
updated